scripts/receipts.sh

```diff
@@ -104,4 +108,59 @@ function receipts::generate() {
   syft packages "${image}" --output cyclonedx-json --file "${output}"
 }
 
+# Generates syft receipts for each architecture for given oci archives
+function receipts::generate::multi::arch() {
+  local buildArchive runArchive registryPort registryPid localRegistry imageType archiveName imageReceipt
+
+  buildArchive="${1}"
+  runArchive="${2}"
+
+  # This is hard-coded, but a random port could be obtained if needed
+  registryPort=54321
+
+  # Start a local in-memory registry so we can work with the oci archives
+  PORT=$registryPort crane registry serve --insecure > /dev/null 2>&1 &
+  registryPid=$!
+
+  sleep 2
```

Comment:
Why do we need to sleep?

Reply:
I added that to give the local registry enough time to start up (in the background). We could change it to a loop where we curl the endpoint until it's ready, but I was going for speed so I went with sleep.